test/unit/rise-twitter.spec.js

```diff
@@ -1,10 +1,84 @@
 import RiseTwitter from "../../src/rise-twitter";
 
-describe("Example test", () => {
-  it("works", () => {
-    expect(RiseTwitter).toBeDefined();
+let component = null;
+let risePlaylistItem = null;
+describe("Twitter Component - Unit", () => {
+  beforeAll(() => {
+    risePlaylistItem = document.createElement("rise-playlist-item");
+    document.getElementsByTagName("body")[0].appendChild(risePlaylistItem);
+    component = new RiseTwitter();
+
+    component._play = jest.genMockFn();
+    component._pause = jest.genMockFn();
+    component._stop = jest.genMockFn();
+  });
 
-    const component = new RiseTwitter();
+  it("should have component defined", () => {
+    expect(RiseTwitter).toBeDefined();
     expect(component).toBeDefined();
   });
+
+  it("should call configure listener when configure event is dispached", (done) => {
+    jest.useFakeTimers();
+
+    const event = new CustomEvent("configure", {
+      detail: {
+        screenName: "test"
+      }
+    });
+
+    risePlaylistItem.dispatchEvent(event);
+
+    setTimeout(()=>{
+      expect(component.screenName).toEqual("test");
+      done();
+    }, 1000);
+
+    jest.runAllTimers();
+  });
+
```

Comment:
@rodrigopavezi Small catch - all three of the following tests only test for `play` event.

Reply:
@jungeunyoon ouch :sob: I will change. Thanks for the catch.

Reply:
@rodrigopavezi haha teamwork. looks good now 👍